README.md

```diff
@@ -39,22 +43,36 @@ use serde_json::json;
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    // Autoconfigure the client from the environment variables AXIOM_TOKEN
+    // and AXIOM_ORG_ID (if it's a personal token):
     let client = Client::new()?;
 
+    // Alternatively build your client by providing the values directly:
+    let client = Client::builder()
+        .with_token("my-token")
+        .with_org_id("my-org")
+        .build()?;
```

Comment:
I'm not 100% sure how I feel about this being in the same file/example? Could people copy this without reading the comments in run into issues running this snippet?

Will it run without errors/warnings? Not to much into Rust (as you might know, lol) but I feel like it _could_ warn on the `client` assigned in line 48 not being used.

Reply:
You get an unused variable warning but it works (it's even tested on CI, check https://github.com/axiomhq/axiom-rs/actions/runs/3540099979/jobs/5942758622, scroll all the way down to ReadmeDoctests)